src/main/java/bio/terra/service/snapshotbuilder/SnapshotBuilderService.java

```diff
@@ -156,8 +191,7 @@ private EnumerateSnapshotAccessRequest convertToEnumerateModel(
               .status(response.getStatus())
               .createdDate(response.getCreatedDate())
               .name(response.getSnapshotName())
-              .researchPurpose(response.getSnapshotResearchPurpose())
-              .createdBy(response.getCreatedBy()));
```

Comment:
why remove createdBy?

Reply:
Ah, I'll re-add that. There was a red squiggle for some reason and I assumed it had been removed, but I had wanted to double check that.